### PR TITLE
Add `circuit-board` icon

### PR DESCRIPTION
**Pull request by danielbayley**
Like from [codicons](https://microsoft.github.io/vscode-codicons/dist/codicon.html)…

### CODE REVIEW

**Comment by karsa-mistmere** [comment]
It's nice, but a bit heavy in comparison to `square`. 🤔 
We don't have it in writing yet, but in #650 we've drafted that icons should aim to have a similar visual weight.

**Comment by danielbayley** [comment]
> It's nice, but a bit heavy in comparison to `square`.

I did try to get it into `square`… But there just isn’t sufficient space for 2 _vias_ without violating the _2px between elements_ rule.

**Comment by jguddas** [comment]
Something like icon or icon (1) would work in the smaller square, with 2 wires and the 2px rule.

**Comment by danielbayley** [comment]
> Something like icon or icon (1) would work in the smaller square, with 2 wires and the 2px rule.

I really like the second one actually…

File: icons/circuit-board.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M5 3h14c1.1 0 2 .9 2 2v14c0 1.1-.9 2-2 2H5c-1.1 0-2-.9-2-2V5c0-1.1.9-2 2-2z" />
+  <circle cx="9" cy="9" r="2" />
+  <path d="M11 9h4c1.1 0 2-.9 2-2V3" />
+  <circle cx="14.5" cy="15" r="2" />
+  <path d="M7 21v-4c0-1.1.9-2 2-2h3.5" />
```
Comment (jguddas) [inline icons/circuit-board.svg:16]:
```suggestion
  <circle cx="14.5" cy="15" r="2" />
  <circle cx="9" cy="9" r="2" />
  <rect x="3" y="3" width="18" height="18" rx="2" ry="2" />
  <path d="M11 9h4a2 2 0 0 0 2-2V3" />
  <path d="M7 21v-4a2 2 0 0 1 2-2h3.5" />
```

**Review by karsa-mistmere — APPROVED** [review]

**Review by ericfennis — CHANGES_REQUESTED** [review]
Some elements are not correctly aligned on the pixel grid.
image

**Comment by danielbayley** [comment]
> Some elements are not correctly aligned on the pixel grid. image

Not sure why exactly, but Illustrator keeps randomly adding decimal values to things if/when I paste between VS Code… So annoying! Fixed anyway…

**Review by ericfennis — APPROVED** [review]
Nice work!

**Pull request merged**